### PR TITLE
Add More Data Capabilities

### PR DESCRIPTION
**Pull request by jamesbsilva**
This is just the fine-tuning ie none of the more detailed eval stuff

* Adds a dataloader for yt_thumbnails with a given file structure and a parquet of label information.

* Changes loss for multi-label

* Makes parameters less memory intensive as suggested in googles README for finetuning. (360, 300) is just chosen to be smaller than full on size

### CODE REVIEW

**Review by zexuan-zhou — APPROVED** [review]
lgtm. BCE loss with imputation of 0 for missing labels

File: bit_pytorch/train.py [hunk 1]
```diff
@@ -17,271 +17,424 @@
 #!/usr/bin/env python3
```
Comment (jamesbsilva) [inline bit_pytorch/train.py:17]: I ran `black` on this file so it has whitespace changes
